trending_data/src/main/webapp/index.html

```diff
@@ -2,18 +2,21 @@
 <html>
   <head>
     <title>Trend Map</title>
-    <script type='module' src="read_maps_key.js"></script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
     <link rel="stylesheet" href="./map_style.css">
+    <script type='module' src="read_maps_key.js"></script>
     <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <script src="https://unpkg.com/@google/markerclustererplus@4.0.1/dist/markerclustererplus.min.js"></script>
     <script src="https://platform.twitter.com/widgets.js" id="twitter-wjs"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
   </head>
   <body>
     <div id="container">
       <div id="map"></div>
       <div id="tweets-container"></div>
     </div>
+
+    <!-- bootstrap scripts -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
So are these actually used? Asking because you mentioned not using bootstrap in the end.

Reply:
Yes these are used, I wasn't able to use [bootstrap-select](https://developer.snapappointments.com/bootstrap-select/) to automatically style the dropdown